src/main/java/GenericsAndCllections/Order.java

```diff
@@ -0,0 +1,45 @@
+package GenericsAndCllections;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Order {
+    public long orderId;
+    private LocalDate date;
+    public User user;
+    public List<Item> items = new ArrayList<>();
+
+    public Order(User user) {
+        Random rand = new Random();
+        this.orderId = rand.nextInt(9999);
+        this.user = user;
+        this.date = LocalDate.now();
+    }
+
+    public Order(List<User> users) {
```

Comment:
This constructor is not used. Did you mean do make an order with a list of items maybe?

Reply:
done